tests/testthat/test-board_url.R

```diff
@@ -176,7 +176,7 @@ test_that("use cache with Last-Modified header", {
   url <- "https://httpbin.org/cache"
 
   expect_condition(
-    http_download(url, path, "test"),
+    http_download(url, path, "test", add_headers = httr::add_headers(x = 1)),
```

Comment:
This is not extensive testing of the new argument, but I don't see a good way forward to really test authentication. Maybe in the Connect tests? Is this worth it?